packages/storage/src/providers/s3/types/options.ts

```diff
@@ -15,12 +15,22 @@ import {
 	StorageSubpathStrategy,
 } from '../../../types/options';
 
+/**
+ * @internal
+ */
+export type AWSTemporaryCredentials = Required<
+	Pick<
+		AWSCredentials,
+		'accessKeyId' | 'secretAccessKey' | 'sessionToken' | 'expiration'
+	>
+>;
```

Comment:
This can be simplified to:
```suggestion
export interface AWSTemporaryCredentials extends Required<AWSCredentials> {}
```

Reply:
Will adress this separately.